app/sidekiq/evss/disability_compensation_form/form526_document_upload_failure_email.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require 'va_notify/service'
+
+module EVSS
+  module DisabilityCompensationForm
+    class Form526DocumentUploadFailureEmail < Job
+      STATSD_SENT_METRIC_KEY = 'api.form_526.document_upload_failure_notification_sent'
+      STATSD_EXHAUSTED_METRIC_KEY = 'api.form_526.document_upload_failure_email_job_exhausted'
+
+      # retry for one day
+      sidekiq_options retry: 14
+
+      sidekiq_retries_exhausted do |msg, _ex|
+        job_id = msg['jid']
+        error_class = msg['error_class']
+        error_message = msg['error_message']
+        timestamp = Time.now.utc
+        form526_submission_id = msg['args'][0]
+
+        Rails.logger.warn(
+          'EVSS::DisabilityCompensationForm::Form526DocumentUploadFailureEmail retries exhausted',
+          { job_id:, error_class:, error_message:, timestamp:, form526_submission_id: }
+        )
+
+        StatsD.increment(STATSD_EXHAUSTED_METRIC_KEY)
+      rescue => e
+        ::Rails.logger.error(
+          'Failure in DisabilityCompensationForm::Form526DocumentUploadFailureEmail#sidekiq_retries_exhausted',
+          {
+            messaged_content: e.message,
+            job_id:,
+            submission_id: form526_submission_id,
+            pre_exhaustion_failure: {
+              error_class:,
+              error_message:
+            }
+          }
+        )
+        raise e
+      end
+
+      def perform(form526_submission_id, supporting_evidence_attachment_guid)
+        @notify_client ||= VaNotify::Service.new(Settings.vanotify.services.benefits_disability.api_key)
+        submission = Form526Submission.find(form526_submission_id)
+
+        email_address = submission.get_email_address
+        first_name = submission.get_first_name
+        date_submitted = submission.get_formatted_creation_time
+
+        template_id = Settings.vanotify.services
+                              .benefits_disability.template_id.form526_document_upload_failure_notification_template_id
+
+        form_attachment = SupportingEvidenceAttachment.find_by!(guid: supporting_evidence_attachment_guid)
+        # We need to obscure the original filename since it may contain PII
+        obscured_filename = form_attachment.obscured_filename
+
+        @notify_client.send_email(
+          email_address:,
+          template_id:,
+          personalisation: {
+            first_name:,
+            filename: obscured_filename,
+            date_submitted:
+          }
+        )
+
+        Rails.logger.info(
```

Comment:
Should this be logger warn or info? We seem to use both. I'm guessing we don't want to use the warn level here since we are already warning in `SubmitUploads` that that job exhausted. This is just logging the fact we sent the email

Reply:
If you only want logging details to verify that emails are being sent, `info` fits that purpose. Otherwise, the use of `warn` should be used to indicate something potentially problematic, like job exhaustion.

Reply:
Datadog can filter log entries by assigned status/severity. I think `warn` and `error` should be reserved for events requiring attention.